src/IdentityAuthClient.js

```diff
@@ -57,7 +57,13 @@ export default class IdentityAuthClient {
     }
 
     _checkToken(token) {
-        // TODO: Test current API token via /account endpoint
+        // TODO: Test current API token via `/account` endpoint
+        // At the moment, Sydent doesn't implement `/account`, so we can't use
+        // that yet. We could try a lookup for a null address perhaps...?
+        // In any case, we should ensure the token in `localStorage` is cleared
+        // appropriately. We already clear storage on sign out, but we'll need
```

Comment:
probably another maintenance issue for when sydent supports the endpoint. It's good practice to assume the token will fail all the time, so we should be checking it. 

Likewise, we should have an issue to explicitly call logout on the IS/IM if we don't already have that issue.

Reply:
ftr the only reason I keep arguing for issues is that it eliminates the excuse of "oops, we forgot" because the issue should fall into some bucket of tracked work.

Reply:
Filed [token test](https://github.com/vector-im/riot-web/issues/10452), [IM logout](https://github.com/vector-im/riot-web/issues/10453), and [IS logout](https://github.com/vector-im/riot-web/issues/10455).

No need to justify, I agree it's great to track these as issues as soon as we spot them, and I frequently suggest the same when reviewing. 😁

Reply:
under Canadian law I'm required to be sorry for things, sorry.